components/x-live-blog-post/src/LiveBlogPost.jsx

```diff
@@ -70,7 +70,7 @@ const LiveBlogPost = ({
 			/>
 		)
 	}
-	if (typeof byline === 'object' && 'tree' in byline) {
+	if (byline && typeof byline === 'object' && 'tree' in byline) {
```

Comment:
**thought**: this suggests our types aren't up to scratch if typescript didn't mind this 😬

Reply:
i don't think there are any types in x-dash?

Reply:
frick i thought i was in the other repo

Reply:
although curiously I just tried it in our repo and it isn't caught there either.
```
(property) byline?: {
    readonly __typename?: "StructuredContent" | undefined;
    readonly tree: any;
} | undefined
```

That's the generated type, but the data we're getting from the API is:
```
        {
          "byline": null
        }
```

Reply:
frick

Reply:
ah from our codegen config:

```
# make types better-compatible with React, which doesn't like null
      maybeValue: T
```

Reply:
😶